### PR TITLE
update brave-ui to 0.33.x

### PR DESCRIPTION
**Pull request by cezaraugusto**
fix https://github.com/brave/brave-browser/issues/2052
closes https://github.com/brave/brave-browser/issues/2053

Test Plan:

`npm run test-unit`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -221,7 +221,7 @@
       "json"
     ],
     "transform": {
-      ".(ts|tsx)": "ts-jest"
+      ".(jsx|js|ts|tsx)": "ts-jest"
```
Comment (NejcZdovc) [inline package.json]: why do we need to add js in ts transformer?

Comment (cezaraugusto) [reply package.json]: you need to `npm i brave-ui@latest` first to try this.

this line is needed so you can transform jsx/js files from brave-ui. without it tests still fail.

Comment (cezaraugusto) [reply package.json]: btw ts-jest uses babel-jest under the hood. you need it so you can transform es6 modules to commonjs and make jest happy

**Comment by jasonrsadler** [comment]
master (0.59.x): 32c990c

**Comment by cezaraugusto** [comment]
0.58.x uplift request
0.57.x uplift request

asking for uplifting since this blocks every new interface that can land in these releases.

**Comment by kjozwiak** [comment]
Approved uplift to `0.57.x` after deliberating with @srirambv & @rebron 👍

**Pull request merged**